docs/user-guide/configuration.md

```diff
@@ -130,6 +130,52 @@ Globally configure how `dates` should be formatted. All [python's `strftime` dir
 
 Globally configure how `time` should be formatted. All [python's `strftime` directives](http://strftime.org) are supported.
 
+### Default tags
+
+Tags can be automatically added for some projects. This is convenient when the
+same tags are always attached to a project.
+
+These automatically attached tags are defined in the `[default_tags]` section
+of the configuration. Each option bellow that section is a project to which
+tags should be attached.
```

Comment:
I would have written something like:

Default tags are defined in the `[default_tags]` section. Each entry should follow the pattern: `project = tag1 tag2`.